### PR TITLE
Create supunsathsara.json

### PR DESCRIPTION
**Pull request by supunsathsara**
Added my profile

## Changes proposed
added my profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1030"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

File: public/data/supunsathsara.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Supun sathsara",
```
Comment (syedsohan) [inline public/data/supunsathsara.json:2]:
```suggestion
  "name": "Supun Sathsara",
```

File: public/data/supunsathsara.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ssupunsathsara",
+      "icon": "twitter"
+    },
+    {
+      "name": "Check My Website",
+      "url": "https://supunsathsara.com",
+      "icon": "globe"
+    },
```
Comment (syedsohan) [inline public/data/supunsathsara.json:21]:
```suggestion
    }
```

Comment (Panquesito7) [inline public/data/supunsathsara.json:21]:
```suggestion
    }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/supunsathsara.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "Launched my personal Website",
+      "date": "November 2021",
+      "icon": "globe",
+      "color": "black",
+      "description": "www.supunsathsara.com"
```
Comment (syedsohan) [inline public/data/supunsathsara.json:29]:
```suggestion
      "description": "Website link - www.supunsathsara.com"
```

File: public/data/supunsathsara.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "title": "Joined GitHub",
+      "date": "June 2020",
+      "icon": "gitHub",
+      "color": "grey",
+      "description": "@supunsathsara"
```
Comment (syedsohan) [inline public/data/supunsathsara.json:36]:
```suggestion
      "description": "GitHub username - @supunsathsara"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**